src/client.ts

```diff
@@ -785,7 +788,7 @@ export class SpvWalletClient {
   FinalizeTransaction(draftTransaction: DraftTransaction): string {
     if (!this.options?.xPriv) {
       const Err = new Error('cannot sign transaction without an xPriv')
-      logger.error(Err)
+      this.logger.error(Err.message)
```

Comment:
For now, we don't use other methods than `.error()` but for future usage I think it will work perfectly 👍